canonical-web-and-design/ubuntu.com
# Initial version of /appliance/plex/raspberry-pi page

## Done

- created /shared folder with logical, reusable steps for other appliances

## Notes

- will abstract more when we do a second appliance

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/plex/raspberry-pi
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
